### PR TITLE
Bug fix for lat lon

### PR DESCRIPTION
**Pull request by egrace479**
- Checks that latitude is within [-90,90] and longitude is within [-180,180], otherwise labels the outliers as `unknown`. 
- Returns error if either column contains a data type that can't be converted to a float.
- Explanation of this has been added to README (with type and range explanation).
- Closes Issue #48

### CODE REVIEW

File: dashboard.py [hunk 1]
```diff
@@ -97,6 +98,18 @@ def parse_contents(contents, filename):
...
             included_features.append(feature)
     
+    # Check for lat/lon bounds & type if columns exist
+    if mapping:
+        try:
+            # Check lat and lon within appropriate ranges (lat: [-90, 90], lon: [-180, 180])
+            if len(df.loc[np.abs(df['lat'].astype(float)) > 90]) > 0:
+                df.loc[np.abs(df['lat'].astype(float)) > 90, 'lat'] = 'unknown'
+            if len(df.loc[np.abs(df['lon'].astype(float)) > 180]) > 0:
+                df.loc[np.abs(df['lon'].astype(float)) > 180, 'lon'] = 'unknown'
```
[the remark this answers is not in the document]
Comment (egrace479) [reply dashboard.py:108]: Oh, that's much better!

**Comment by johnbradley** [comment]
@egrace479 Since this is a bug fix could you add a unit test? It looks like you added two sample csv files that you could be used to test the logic.

**Review by johnbradley — APPROVED** [review]
LGTM

File: dashboard.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import pandas as pd
+import numpy as np
```
Comment (thompsonmj) [inline dashboard.py:2]: I think this adds a new dependency to include in `requirements.txt`, right?

**Review by thompsonmj — CHANGES_REQUESTED** [review]

File: dashboard.py (hunk 2 above)
Comment (johnbradley) [reply dashboard.py:2]: pandas is built on top of numpy

Comment (egrace479) [reply dashboard.py:2]: No, `numpy` is included in `pandas`.

**Review by thompsonmj — APPROVED** [review]
👍

**Pull request merged**